### PR TITLE
add hapi-wechat & hapi-seneca

### PR DESCRIPTION
**Pull request by dhso**

### CODE REVIEW

File: lib/plugins.js [hunk 1]
```diff
@@ -246,6 +246,10 @@ exports.categories = {
         susie: {
             url: 'https://github.com/mtharrison/susie',
             description: 'Server-Sent Events for hapi with support for streaming events'
+        },
+        hapi-wechat: {
```
Comment (devinivy) [inline lib/plugins.js]: This key needs single quotes (`'`) around it because it contains a dash (`-`).

File: lib/plugins.js [hunk 2]
```diff
@@ -776,6 +780,10 @@ exports.categories = {
         thurston: {
             url: 'https://github.com/ruiquelhas/thurston',
             description: 'Route-level file signature validation for request payload file streams'
+        },
+        hapi-seneca: {
```
Comment (devinivy) [inline lib/plugins.js]: This key needs single quotes (') around it because it contains a dash (-).

File: lib/plugins.js (hunk 1 above)
Comment (dhso) [reply lib/plugins.js]: oh, thanks.

**Comment by Nargonath** [comment]
thanks @devinivy

**Pull request merged**